nautobot_ssot/jobs/__init__.py

```diff
@@ -43,6 +43,8 @@ def get_data_jobs():
     data_sources = []
     data_targets = []
     for job in sync_jobs:
+        if job.job_class is None or not issubclass(job.job_class, (DataSource, DataTarget)):
```

Comment:
```suggestion
        if job.job_class is None:
```

I would argue that this is enough since the `issubclass` check happens right after, right?

Reply:
Agreed, but this check should speed up the process since it eliminates all Jobs before they get to the subclass check.